tests/data/messages/prod/anvir2/G1809522146-ASF.msg

```diff
@@ -1 +1,43 @@
-{"accessToken":"1ho/Miw/djQl0zbiwLCwy8/Dis6ehYpg:HNxIh6UN6uMm0sGDtszTwzvS3lk+S3dMT3PhyENmhyoPYxdSXDp9CKUxEQsHzbAQBRkM2RtFr1bROs2fPsOQ48LBhh5AsGomwYxlRFvmfrY=","callback":"http://172.17.0.1:3001/service/4a9bd2b6-71c2-4430-8561-74175d5540cd","client":"harmony-local","format":{"mime":"image/tiff"},"isSynchronous":true,"requestId":"4a9bd2b6-71c2-4430-8561-74175d5540cd","sources":[{"collection":"C1808440897-ASF","granules":[{"bbox":[32.4875114,12.8803591,33.2936786,13.7245336],"id":"G1809522146-ASF","name":"ALAV2A104483330","temporal":{"end":"2008-01-10T08:25:00.636Z","start":"2008-01-10T08:24:48.953Z"},"url":"https://datapool.asf.alaska.edu/ORI/A3/ALAV2A104483330-OORIRFU.zip"}]}],"stagingLocation":"s3://local-staging-bucket/public/asfdataservices/gdal-subsetter/28a46d50-51c6-4f13-9cf8-c6e6c61119a1/","subset":{"bbox":[32.9,13.2,33.1,13.4]},"user":"cirrusasf","version":"0.9.0"}
+{
```

Comment:
I made a couple of the message files indented JSON, for easier reading. The content has remained the same.